ios/browser/api/bookmarks/importer/favicon_reencode.mm

```diff
@@ -12,7 +12,7 @@
 
 namespace gfx {
   //FROM: ui/gfx/geometry/size.h
-  const int kFaviconSize = 16;
+constexpr int kFaviconSize = 16;
```

Comment:
nit: index is wrong

Reply:
Sorry, I'm not following what you are saying...

Reply:
There should be two spaces indention just like the comment above.